From ba6cba9eec00f222223cc5b7e31e7332c7e372af Mon Sep 17 00:00:00 2001
From: Markus Trippelsdorf <markus@trippelsdorf.de>
Date: Mon, 13 Aug 2012 21:24:17 +0200
Subject: [PATCH] Update x86-64 ULPs

The recent clog and clog10 fixes are causing some failing tests on my
AMD64 CPU.
---
 sysdeps/x86_64/fpu/libm-test-ulps | 3 +++
 2 files changed, 7 insertions(+)

diff --git a/sysdeps/x86_64/fpu/libm-test-ulps b/sysdeps/x86_64/fpu/libm-test-ulps
index cdf80c3..8a38221 100644
--- a/sysdeps/x86_64/fpu/libm-test-ulps
+++ b/sysdeps/x86_64/fpu/libm-test-ulps
@@ -817,6 +817,9 @@ float: 1
 ifloat: 1
 ildouble: 1
 ldouble: 1
+Test "Real part of: clog (0x1.234566p-30 + 1.0 i) == 5.614163921211322622623353961365728040115e-19 + 1.570796325735258575254858696548386439740 i":
+ildouble: 1
+ldouble: 1
 Test "Real part of: clog (0x1.fffffep+127 + 0x1.fffffep+127 i) == 89.06941264234832570836679262104313101776 + pi/4 i":
 ildouble: 1
 ldouble: 1
-- 
1.7.12

